lib/gmsmith.js

```diff
@@ -134,8 +134,14 @@ engine.createCanvas = createCanvas;
  */
 function createImage(file, cb) {
   // Create the image
-  var gm = getGm(),
-      img = gm(file);
+  var gm = getGm();
+  var img;
+
+  if (file.buffer) {
+    img = gm(file.buffer, file.id);
```

Comment:
While this works for getting image info, it doesn't work with `gm` when generating the spritesheet itself. In that case, we are currently passing through the filepath to `gm`:

https://github.com/twolfson/gmsmith/blob/0.5.0/lib/gmsmith.js

I have no idea how we can delimit different images via `stdin` to `gm` =/